apps/nowcasting-app/components/utils.ts

```diff
@@ -42,16 +42,28 @@ export const convertISODateStringToLondonTime = (date: string) => {
   return date_london_time_str;
 };
 
+
 export const formatISODateStringHuman = (date: string) => {
   // Change date to nice human readable format.
   // Note that this converts the string to Europe London Time
   // timezone and seconds are removed
+
   const d = new Date(date);
-  const date_london = d.toLocaleDateString("en-GB", { timeZone: "Europe/London" });
-  const date_london_time = d.toLocaleTimeString("en-GB", { timeZone: "Europe/London" }).slice(0, 5);
+  
+  const options = {
```

Comment:
to solve the vercel error do you need to use " not 's? 

or does it just not like youve entered a dictoinary instead of writing it out

Reply:
thanks for the feedback. I just tried writing it out, and it seems to work!